continuous_integration/recipes/distributed/conda_build_config.yaml

```diff
@@ -1,3 +1,6 @@
 python:
     - 3.8
     - 3.9
+build_ext:
+    - "cython"
```

Comment:
Curious as to what this would do - since we are already building `linux-64` packages, would this just stop the cython builds from being converted to non-Linux platforms?

Reply:
Cool, just interested in the impact - I'll go ahead and leave this selector here on the off-chance that we ever begin running builds on a non-Linux platform